servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsClient.java

```diff
@@ -254,9 +255,10 @@ protected AbstractDnsSubscription newSubscription(
                                 } else {
                                     final DnsAnswer<HostAndPort> dnsAnswer;
                                     long minTTLSeconds = Long.MAX_VALUE;
+                                    List<DnsRecord> dnsRecords = null;
                                     try {
-                                        @SuppressWarnings("unchecked")
-                                        final List<DnsRecord> dnsRecords = (List<DnsRecord>) completedFuture.getNow();
+                                        //noinspection unchecked
```

Comment:
We have been favoring annotations for suppression instead of comments.

Reply:
Annotation is allowed only when you define a new local variable. In this case, when I reassign a value, it's not allowed.

Reply:
Is this comment IDE specific? if so consider keeping the local variable to preserve the previous `@ SuppressWarnings` and re-assign to outer scope variable.